packages/server/test/support/fixtures/projects/hooks-after-rerun/cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
```

Comment:
you should set `supportFile: false` in the cypress config and delete these files. https://github.com/cypress-io/cypress/blob/2-fix-rerun-hooks-on-navigation/packages/server/test/support/fixtures/projects/hooks-after-rerun/cypress.json